### ISSUE TITLE
Error using radio receivedNumber/String if variable is already declared

### ISSUE THREAD

**Issue by microbit-mark** [Microsoft/pxt-microbit#627]
microbit-support:3169

If you start your program with an onRadioReceived receivedNumber, and then use the drop down menu to change receivedNumber to receivedString, the variable type is not changed in the JS code, but it appears that it has and results in an operator error.
![image](https://user-images.githubusercontent.com/31242877/35872133-f0899354-0b5d-11e8-86aa-7fcdb1256720.png)

https://makecode.microbit.org/51330-19337-18540-12529

This can only happen if you have already declared receivedString as a variable, but in the classroom it is common practice to start with and extend a previous program/project, so it's becomes a common occurrence.

From @pelikhan: We need to remove the option to rename those mutator variables.

**Comment by pelikhan** [comment]
```
radio.onDataPacketReceived( ({ receivedNumber: receivedString }) =>  {
    if (receivedString == "hi") {
    	
    }
})
radio.setGroup(9)
```

**Comment by pelikhan** [comment]
The ``delete`` button also should be removed.

**Comment by pelikhan** [comment]
Fix in v0.14.22.
